lib/spdy-transport/connection.js

```diff
@@ -281,7 +281,7 @@ Connection.prototype._handleFrame = function _handleFrame(frame) {
 
   // Session window update
   if (frame.type === 'WINDOW_UPDATE' && frame.id === 0) {
-    if (state.version < 3.1 && state.autoSpdy31) {
```

Comment:
Is there a way to get both SPDY/3 and SPDY/3.1 clients to work in plain mode? In its current form, the autoSpdy31 flag causes the library to be purely binary with respect to v3 and v3.1. Its all or nothing.